### DISCUSSION TITLE
[Feature suggestion] - Option to subscribe and get notifications about posts to other threads in a discussion

### DISCUSSION THREAD

**Discussion by marcr** [mastodon/mastodon#24307] [General]
Hi all,

It often happens that when I reply to a post, that I get into a dialogue with the person who started the thread. Others may also join in, but that's not the point. I get notifications for the thread where I'm a part of, but not for other threads in the discussion.

So you may find yourself discussing the same thing several times, because you don't know if there are other threads with new posts. The only way to find out, is to keep clicking on the first post and scroll through all the replies.

I would like to be able to subscribe to a thread or discussion and be notified of new replies below the subscribed post.
That is, I'd like to be able to do this not just for the very first post, but for any post, marking it as a sort of root post for all further threads I'm interested in.

I've looked around to see if this is already possible, but haven't found anything in the settings. And I'm not sure if this is technically feasible, because I'm obviously not mentioned in the other threads.
I also know that due to federation, not all replies may be visible on my instance, and to see them all I would have to look at the original post, but that wouldn't negate the usefulness of the suggestion in my eyes.

What do you think?

Best regards
Marc

**Comment by Mart-Bogdan** [comment]
Would be nice.

**Comment by dreua** [comment]
Related issue: https://github.com/mastodon/mastodon/issues/7969
